### YamlMime:ManagedReference
items:
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  id: RuntimeBinderException
  children:
  - Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor
  - Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String)
  - Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: RuntimeBinderException
  nameWithType: RuntimeBinderException
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  type: Class
  summary: "Vertegenwoordigt een fout die optreedt wanneer een dynamische binding in de C# runtime binder wordt verwerkt."
  remarks: "Uitzonderingen van dit type verschillen van <xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException>in dat RuntimeBinderException duidt op een fout in de zin van een gebruikelijke compilatiefout binden terwijl <xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException>vertegenwoordigt een defect van de runtime-binder zelf.</xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException> </xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException>"
  syntax:
    content: 'public class RuntimeBinderException : Exception'
  inheritance:
  - System.Exception
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor
  id: '#ctor'
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  langs:
  - csharp
  name: RuntimeBinderException()
  nameWithType: RuntimeBinderException.RuntimeBinderException()
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException()
  type: Constructor
  assemblies:
  - Microsoft.CSharp
  namespace: Microsoft.CSharp.RuntimeBinder
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&quot;> </xref> klasse."
  remarks: "Deze constructor initialiseert de <xref:System.Exception.Message%2A>eigenschap van het nieuwe exemplaar voor een systeem geleverd bericht met een beschrijving van de fout, zoals &quot;DefaultMessageDisplayedByParameterlessCtorWriterMustSupply&quot; Dit bericht rekening gehouden met de huidige systeemcultuur.</xref:System.Exception.Message%2A>       De volgende tabel toont de eerste waarden voor een exemplaar van <xref:System.InsufficientMemoryException>.</xref:System.InsufficientMemoryException>      | Eigenschap | Waarde |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| De gelokaliseerde fouttekenreeks bericht. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public RuntimeBinderException ();
    parameters: []
  overload: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  langs:
  - csharp
  name: RuntimeBinderException(String)
  nameWithType: RuntimeBinderException.RuntimeBinderException(String)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(String)
  type: Constructor
  assemblies:
  - Microsoft.CSharp
  namespace: Microsoft.CSharp.RuntimeBinder
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&quot;> </xref> klasse met een opgegeven foutbericht."
  remarks: "De volgende tabel toont de eerste waarden voor een exemplaar van <xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException>.</xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException>      | Eigenschap | Waarde |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| De fouttekenreeks bericht opgegeven in `message`. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public RuntimeBinderException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Het bericht dat de uitzondering wordt beschreven. De aanroeper van deze constructor is vereist om ervoor te zorgen dat deze tekenreeks gelokaliseerd voor de huidige systeemcultuur."
  overload: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  langs:
  - csharp
  name: RuntimeBinderException(SerializationInfo,StreamingContext)
  nameWithType: RuntimeBinderException.RuntimeBinderException(SerializationInfo,StreamingContext)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - Microsoft.CSharp
  namespace: Microsoft.CSharp.RuntimeBinder
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&quot;> </xref> klasse die gegevens is geserialiseerd."
  remarks: "Deze constructor is aangeroepen tijdens de deserialisatie op het object exception verzonden via een stroom opnieuw samenstellen."
  syntax:
    content: protected RuntimeBinderException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Het object dat de geserialiseerde objectgegevens over de uitzondering bevat..."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "De contextuele informatie over de bron- of doelserver."
  overload: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  langs:
  - csharp
  name: RuntimeBinderException(String,Exception)
  nameWithType: RuntimeBinderException.RuntimeBinderException(String,Exception)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(String,Exception)
  type: Constructor
  assemblies:
  - Microsoft.CSharp
  namespace: Microsoft.CSharp.RuntimeBinder
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&quot;> </xref> klasse die een opgegeven foutmelding en een verwijzing naar de interne uitzondering dat de oorzaak van deze uitzondering heeft."
  remarks: "Een uitzondering die is gegenereerd als direct resultaat van een eerdere uitzondering moet een verwijzing bevatten naar de vorige uitzondering in de <xref:System.Exception.InnerException%2A>eigenschap.</xref:System.Exception.InnerException%2A> De <xref:System.Exception.InnerException%2A>eigenschap dezelfde waarde die is doorgegeven aan de constructor of `null` als de <xref:System.Exception.InnerException%2A>eigenschap levert niet de waarde van de binnenste uitzondering voor de constructor.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       De volgende tabel toont de eerste waarden voor een exemplaar van <xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException>.</xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException>      | Eigenschap | Waarde |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| De fouttekenreeks bericht opgegeven in `message`. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public RuntimeBinderException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Het foutbericht met uitleg over de reden voor de uitzondering."
    - id: innerException
      type: System.Exception
      description: "De uitzondering is de oorzaak van de huidige uitzondering of een null-verwijzing als geen binnenste uitzondering is opgegeven."
  overload: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException()
  nameWithType: RuntimeBinderException.RuntimeBinderException()
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException()
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String)
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException(String)
  nameWithType: RuntimeBinderException.RuntimeBinderException(String)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException(SerializationInfo,StreamingContext)
  nameWithType: RuntimeBinderException.RuntimeBinderException(SerializationInfo,StreamingContext)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException(String,Exception)
  nameWithType: RuntimeBinderException.RuntimeBinderException(String,Exception)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(String,Exception)
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException
  nameWithType: RuntimeBinderException.RuntimeBinderException
